conf/planned-downtime.conf

```diff
@@ -23,8 +23,8 @@
 #    downtime should only be removed from this config after 30 days since the planned downtime date
 
 planned-downtime = [
-  {"startDate": "2024-06-08", "startTime": "22:00", "endDate": "2024-06-08", "endTime": "23:00", "affectedChannel": "GB Arrivals"},
-  {"startDate": "2024-06-08", "startTime": "22:00", "endDate": "2024-06-08", "endTime": "23:00", "affectedChannel": "GB Departures"},
-  {"startDate": "2023-09-05", "startTime": "22:00", "endDate": "2023-09-05", "endTime": "23:00", "affectedChannel": "XI Arrivals"},
-  {"startDate": "2023-09-05", "startTime": "22:00", "endDate": "2023-09-05", "endTime": "23:00", "affectedChannel": "XI Departures"}
+  {"startDate": "2024-07-01", "startTime": "06:00", "endDate": "2024-07-01", "endTime": "13:00", "affectedChannel": "GB Arrivals"},
```

Comment:
I'm just looking at that comment above - I don't know if there's a technical reason for that, but with that in mind do you think it's safest to leave that 2024-06-08 entry in here for now?